src/varity/ref_gene.clj

```diff
@@ -193,14 +201,14 @@
                   :strand strand))))
 
 (defn load-gencode
-  [f parse-line]
+  [f parse-line & {:keys [chunk-size] :or {chunk-size 10000}}]
```

Comment:
It might be better to add this optional arg to both `load-gtf` and `load-gff` because currently these functions are easier to use. Or reimplement `load-gencode` to pick up an appropriate function based on `f`'s extension (sorry my first implementation is not well designed).

Reply:
Thanks. I had missed it.

BTW, I think the GTF/GFF3 loader should use cljam's implementation (https://github.com/chrovis/cljam/blob/master/src/cljam/io/gff.clj). I am willing to do a complete rewrite on that.